src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.model.dto.request;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.PasswordMatch;
+
+@PasswordMatch
+public class UserRequestDto {
+    @Email
+    @NotNull(message = "This field is obligatory")
+    private String email;
+    @NotNull(message = "This field is obligatory")
+    @NotBlank
```

Comment:
Do we need to use both annotations @NotNull and @NotBlank ?

Reply:
No, because I used @Size(length=4). If I didn't use size annotation then it would make sense, but right now no need in @notblank

Reply:
Annotation @NotBlank already includes NotNull: "The annotated element must not be null and must contain at least one non-whitespace character."